common-modules/src/main/java/com/softserve/if072/common/model/RestResponsePage.java

```diff
@@ -0,0 +1,130 @@
+package com.softserve.if072.common.model;
+
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Nazar Vynnyk
+ */
+public class RestResponsePage<T> extends PageImpl<T> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)